css/css-grid-1/alignment/grid-content-alignment-004.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Grid Layout Test: Content Distribution 'stretch'</title>
+<link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-align-3/#content-distribution">
+<link rel="help" href="https://drafts.csswg.org/css-align-3/#valdef-align-content-stretch">
+<link rel="match" href="../reference/grid-content-alignment-stretch-2x2-ref.html">
+<meta name="assert" content="Content Distribution properties render correcly given value of 'stretch'.">
+<style>
+.grid {
+  display: grid;
+  background: grey;
+  width: 200px;
+  height: 200px;
+  grid-template-columns: auto auto;
+  grid-template-rows: auto auto;
+  align-content: stretch;
```

Comment:
Nit: I'd add 1 more test `*-005.html` to verify that the default alignment is `stretch` too.